main.go

```diff
@@ -31,6 +32,13 @@ import (
 	"github.com/metalmatze/signal/healthcheck"
 	"github.com/metalmatze/signal/internalserver"
 	"github.com/metalmatze/signal/server/signalhttp"
+	logsv1 "github.com/observatorium/api/api/logs/v1"
```

Comment:
Convention is to keep all of the imports grouped so that all Observatorium/API packages are together at the bottom